npm/pkg/controlplane/controllers/v2/podcontroller.go

```diff
@@ -605,18 +596,17 @@ func (c *PodController) manageNamedPortIpsets(portList []corev1.ContainerPort, p
 			protocol = fmt.Sprintf("%s:", port.Protocol)
 		}
 
-		namedPort := util.NamedPortIPSetPrefix + port.Name
 		namedPortIpsetEntry := fmt.Sprintf("%s,%s%d", podIP, protocol, port.ContainerPort)
 
 		// nodename in NewPodMetadata is nil so UpdatePod is ignored
-		podMetadata := dataplane.NewPodMetadata(podKey, namedPortIpsetEntry, "")
+		podMetadata := dataplane.NewPodMetadata(podKey, namedPortIpsetEntry, nodeName)
```

Comment:
Curious - why do we have `NodeName`? Is it because of `endpointPolicy` of windows?

Reply:
Yes, in linux this will be no-op